### ISSUE TITLE
Page not found error when clicking on cards

### ISSUE THREAD

**Issue by 9a3eedi** [wekan/wekan#1405]
## Issue
**Impacted version**: 0.62-3-g74877fd

**Server Setup Information**:

* Operating System: Ubuntu Server 16.04 64-bit
* Deployment Method: snap
* Node Version: N/A
* MongoDB Version: N/A
* ROOT_URL environment variable: http://192.168.0.51:80/wekan

**Problem description**:
I'm running the snap version of Wekan, tracking stable. Looking at my server it seems to have upgraded itself yesterday midnight to the latest version (0.62-3-g74877fd), and today I noticed that I cannot view the cards anymore. Clicking on the cards will show me a page not found error. I tried restarting the server but that doesn't help. 

It also doesn't seem to be related to previous issues like #666 or #726 but I may have missed something...

I am still able to create new cards and move existing cards around, but I cannot view them.

**Comment by xet7** [comment]
@9a3eedi

```
sudo snap set wekan root-url="http://192.168.0.51"
sudo snap set wekan port="80"
sudo systemctl restart snap.wekan.wekan
```

Then go with webbrowser to http://192.168.0.51

**Comment by 9a3eedi** [comment]
Thanks, this works.

But is there a simple way to preserve the old URL without having to resort to Apache forwarding?

**Comment by xet7** [comment]
@9a3eedi 

```
sudo snap set wekan root-url="http://192.168.0.51:80/wekan"
sudo snap set wekan port="80"
sudo systemctl restart snap.wekan.wekan
```

**Comment by 9a3eedi** [comment]
@xet7 Following your exact instructions causes the "Page not found" error to happen again.

**Comment by naveengh** [comment]
I have exact same issue.

I have created image using Intuz on AWS to access wekan. Everything looks good so far - can create boards, cards. 

But when I click on card to view details, i get following error and no page to display
Safari can't open the page https://0.0.0.0/wekan/b/Hbwfy79pdPhfwyby9/welcome-board/pkosgAPBHzbHif9mo

When you click on card it takes request to https://0.0.0.0/ which is obviously incorrect so I replaced 0.0.0.0 with AWS public IP and I can see card details.

So where do I insert AWS public IP?

Thanks for help.
